src/gui/PasswordGeneratorWidget.cpp

```diff
@@ -279,5 +281,12 @@ void PasswordGeneratorWidget::updateGenerator()
     m_generator->setCharClasses(classes);
     m_generator->setFlags(flags);
 
+    if (m_generator->isValid()) {
+        m_ui->buttonGenerate->setEnabled(true);
+    }
+    else {
```

Comment:
Although much of the existing code uses this style, it is actually against both the Qt coding standard and my personal taste. I'd suggest removing the newline before else.